flux_led/device.py

```diff
@@ -458,23 +458,24 @@ def setColdWhite255(self, level, persist=True, retry=2):
         self.set_levels(w2=level, persist=persist, retry=retry)
 
     def setWhiteTemperature(self, temperature, brightness, persist=True, retry=2):
-        w, w2 = color_temp_to_white_levels(temperature, brightness)
-        self.set_levels(w=w, w2=w2, persist=persist, retry=retry)
+        cold, warm = color_temp_to_white_levels(temperature, brightness)
+        self.set_levels(w=warm, w2=cold, persist=persist, retry=retry)
 
     def getWhiteTemperature(self):
         # Assume input temperature of between 2700 and 6500 Kelvin, and scale
         # the warm and cold LEDs linearly to provide that
-        warm = self.raw_state.warm_white / 255
-        cold = self.raw_state.cool_white / 255
-        brightness = warm + cold
-        temperature = ((cold / brightness) * (6493 - 2703)) + 2703
-        brightness = round(brightness * 255)
-        temperature = round(temperature)
-        return (temperature, brightness)
+        raw_state = self.raw_state
+        return white_levels_to_color_temp(raw_state.warm_white, raw_state.cool_white)
 
```

Comment:
This was returning a temperature and a brightness.   Now it's just returning the hex values of the whites?

Reply:
Its returning the result of `white_levels_to_color_temp` which is `color_temp, brightness`

I'll unpack the tuple so its more apparent

Reply:
> [972bda5](https://github.com/Danielhiversen/flux_led/commit/972bda5a36d1c26670dab8b00bec751e9e090cff)

reading is hard.  lol.